moduleroot/spec/spec_helper.rb

```diff
@@ -9,6 +9,11 @@
 require 'rspec-puppet-facts'
 include RspecPuppetFacts
 
+add_custom_fact :concat_basedir, '/tmp'
```

Comment:
Would it be useful to have a comment where each fact usually comes from? Like puppetlabs-concat in this case